addons/account/models/account_move.py

```diff
@@ -4402,6 +4402,10 @@ def _add_cash_basis_lines_to_exchange_difference_vals(lines, exchange_diff_move_
             :param lines:                   The account.move.lines to which fix the residual amounts.
             :param exchange_diff_move_vals: The current vals of the exchange difference journal entry.
             '''
+            if self.env['ir.config_parameter'].set_param('account.skip_fx_on_caba_base_lines', False):
```

Comment:
- `set_param` instead of `get_param`?
- If so, 2nd parameter is not required
- Missing `sudo()`?

Reply:
@luisg123v you are quite right.

Regards.

Reply:
Fixed.